src/components/MenuIcon/index.js

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import { CLOSE__MOBILE__MENU } from "../../constants";
```

Comment:
Use single underscore action type name
```suggestion
import { CLOSE_MOBILE_MENU } from "../../constants";
```

Reply:
But I prefer double underscore, for me, it looks cleaner than single underscore. If you really want me to change it I am fine with it

Reply:
I would leave it to you to choose how you want to do it.